src/components/app/index.tsx

```diff
@@ -23,13 +25,31 @@ export default class App extends Component<Props, State> {
     }
   }
 
+  private async getImageData(bitmap: ImageBitmap): Promise<ImageData> {
+    // Make canvas same size as image
+    const canvas = document.createElement('canvas');
+    [canvas.width, canvas.height] = [bitmap.width, bitmap.height];
```

Comment:
I still think this technique is smarter than it is good, and it'd be better as two lines. Happy to be overridden by @developit or @kosamari though.

Reply:
Nah, let’s keep it explicit. I don’t mind. I copy-pasted this from the `example.html` that I copied from my article :3